kotsadm/pkg/apiserver/server.go

```diff
@@ -78,6 +80,7 @@ func Start() {
 	r.Path("/api/v1/upload").Methods("PUT").HandlerFunc(handlers.UploadExistingApp)
 	r.Path("/api/v1/download").Methods("GET").HandlerFunc(handlers.DownloadApp)
 	r.Path("/api/v1/app/{appSlug}/sequence/{sequence}/renderedcontents").Methods("OPTIONS", "GET").HandlerFunc(handlers.GetAppRenderedContents)
+	r.Path("/api/v1/app/{appSlug}/sequence/{sequence}/contents").Methods("OPTIONS", "GET").HandlerFunc(handlers.GetAppContents)
```

Comment:
is this an entirely new route?

Reply:
Yes.  It used to be a GQL route. This is used on the ViewFiles page to render the application contents. I didn't want to put the OCI get/put code in Typescript, so I ported this endpoint to Go /Rest.